dagshub/common/cli.py

```diff
@@ -1,10 +1,19 @@
+import os
 import sys
+from http import HTTPStatus
+
 import click
 
 import dagshub.auth
 from dagshub.common import config
 import dagshub.common.logging
 import logging
+import git
```

Comment:
We don't have GitPython as a dependent package, the user-installed dependencies are listed in [`setup.py`](https://github.com/DagsHub/client/blob/master/setup.py#L37).
You should add it in there.
Keep in mind when setting the version that we support Python 3.7 and up

Reply:
done